example/pulseaudio_sink.py

```diff
@@ -35,10 +36,10 @@
 page.add_item(Slider("Volume").connect(volume.field('volume')))
 page.add_item(Slider("Balance").connect(volume.field('balance'), convert=True))
 page.add_item(Slider("Fade").connect(volume.field('fade'), convert=True))
-page.add_item(ButtonGroup("", [
+page.add_item(ButtonGroup("", cast(Iterable[AbstractButton], [
```

Comment:
No idea why `mypy` wants this now here and not at other places, e.g. `ui_showcase.py`.  Maybe you have an idea how to solve it without the cast.  Looks kind of ugly.

Reply:
MyPy tries to guess the type of that list here, based on the first common ancestor class of the two buttons it finds. Unfortunately it finds the common ancestor `WebDisplayDatapoint[bool]` instead of `AbstractButton`. So it complains that we are passing a list of ``WebDisplayDatapoint[bool]`s instead of a list of `AbstractButtons`. I assume that, without the new `Union` in the parameter type hint, MyPy was smart enough to find that `List[AbstractButton]`, as expected by the parameter, is also a valid type of that list, so it used that instead of trying to guess the type.

I think, the "correct" solution would be to explicitly type-hint the list:
```python
buttons: List[AbstractButton] = [
    ToggleButton(icon('volume mute')).connect(mute),
    DisplayButton(label=icon('power off')).connect(active),
]
page.add_item(ButtonGroup("", buttons))
```

However, that also looks kind of ugly, especially in the example code. So maybe, we go for a simple type-ignore with a comment, like `# type: ignore  # MyPy has problems with guessing the correct type of the list of buttons`.
Maybe, MyPy will learn this type inference eventually and we can remove the comment. (I will check that once in a while by running MyPy with `--warn-unused-ignores` locally.)

Reply:
Done.  Did only add the `# type: ignore`
Not the reason: `# MyPy has problems with guessing the correct type of the list of buttons.`
I don't think we need this in the example code.  May only be confusing for newbies.